src/core/tritonserver.cc

```diff
@@ -1093,69 +1092,66 @@ TRITONSERVER_ServerModelMetadata(
       lserver->ModelReadyVersions(model_name, &ready_versions));
 
   rapidjson::Document metadata;
+  auto& allocator = metadata.GetAllocator();
   metadata.SetObject();
   // Just store string reference in JSON object since it will be serialized to
   // another buffer.
-  metadata.AddMember(
-      "name", rapidjson::StringRef(model_name), metadata.GetAllocator());
+  metadata.AddMember("name", rapidjson::StringRef(model_name), allocator);
 
   rapidjson::Value versions(rapidjson::kArrayType);
   for (const auto v : ready_versions) {
     auto version_str = std::to_string(v);
-    versions.PushBack(
-        rapidjson::Value(version_str.c_str(), version_str.size()),
-        metadata.GetAllocator());
+    rapidjson::Value version_val(version_str.c_str(), allocator);
+    versions.PushBack(version_val, allocator);
```

Comment:
@deadeyegoodwin according to rapidjson's doc, it is incorrect to use temporary value in `PushBack()`, or you need to invoke the move schema explicitly via `Move()`. The change here is okay and it implicitly uses the move schema.